### ISSUE TITLE
run issue

### ISSUE THREAD

**Issue by Diane0323** [jocpae/clDice#3]
Hello, your pytorch version cldice cannot run successfully. Can you republish a pytorch version cldice in your free time?

**Comment by jocpae** [comment]
Hi, can you please share your code/snippet so we can reproduce your error? or can you provide us the exact error?

**Comment by Diane0323** [comment]
> Hi, can you please share your code/snippet so we can reproduce your error? or can you provide us the exact error?

Hello, when I run the pytorch version, I get a lot of negative numbers. I think it's because I didn't use softmax to normalize y_pred. The current loss performance looks okay, but the training has not been completed yet, thank you very much for this job.
